### ISSUE TITLE
Developer: Active Navigation

### ISSUE THREAD

**Issue by poofichu** [punchcard-cms/punchcard#95]
_As a_ developer _I want_ to implement an active state to top level navigation _so that_ the user knows what section they are in

```gherkin
Feature: Active top level navigation
  As a developer
  I want an active state on top level navigation
  So that the user will know what section they are editing

  Scenario: a developer needs to add top level active navigation
    Given there is an active state to top level navigation
    When a user needs to edit/add content to the CMS
    Then they will be able to see where they are at
```

---

**Comment by poofichu** [comment]
Rachel White [3:12 PM]  
look up how to deal with long content in table cells on responsive

Rachel White [3:17 PM]  
explore navigation menu items

[3:18]  
active tab on that page.

[3:19]  
explore nunjucks templating for knowing active links

**Comment by poofichu** [comment]
Responsive Tables Solution!!
http://codepen.io/geoffyuen/pen/FCBEg

**Comment by poofichu** [comment]
Thinking about this for responsive small screen menu. 
https://codepen.io/andornagy/pen/RNeydj?editors=1100#0
Thoughts @Snugug @rachelnicole ??

**Comment by scottnath** [comment]
@poofichu is this complete? Can this story be closed?

**Comment by poofichu** [comment]
No sir @scottnath. This hasn't been worked on yet.  It is not 100% needed for MVP, but is one of those things that is nice to have.  All other products in the universe have active navigation, so it would be nice enough to have it for this :)

**Comment by joshkimmell** [comment]
@poofichu are these possible scenarios that need to be added to this story?
 - [ ] look up how to deal with long content in table cells on responsive
 - [ ] explore navigation menu items
 - [ ] active tab on that page.
 - [ ] explore nunjucks templating for knowing active links

**Comment by poofichu** [comment]
@joshkimmell that post above was a brain dump so we didn't lose other things that needed to be done.  Please ignore....

The only thing needed for this story is the last item.

**Comment by ineedsubstance** [comment]
@joshkimmell + @scottnath - update this to a user story. Can you guys please review.

@poofichu - can you make sure I didn't lose any part of what you were trying to say.

**Comment by joshkimmell** [comment]
I'm good with this. I think we can move it to the backlog.

**Comment by poofichu** [comment]
Bueno!

**Comment by ineedsubstance** [comment]
🎱

**Comment by joshkimmell** [comment]
🎱 🎱

**Comment by joshkimmell** [comment]
@scottnath 
This will require a FED to work on this. Is this story complete IYO?

**Comment by ineedsubstance** [comment]
Blocked by #570
